crates/api_models/src/payment_methods.rs

```diff
@@ -437,6 +437,7 @@ pub struct PaymentMethodListResponse {
         }
     ]
     ))]
+    pub merchant_name: Option<Encryptable<masking::Secret<String>>>,
```

Comment:
you can use this type alias 
https://github.com/juspay/hyperswitch/blob/6f1a1a3bcb7466738b018a773a5fe4adc5cf2887/crates/common_utils/src/crypto.rs#L444